korangar/src/world/model/mod.rs

```diff
@@ -34,7 +36,10 @@ impl Model {
         let rotation_matrix = Matrix4::from_angle_z(-transform.rotation.z)
             * Matrix4::from_angle_x(-transform.rotation.x)
             * Matrix4::from_angle_y(transform.rotation.y);
-        let scale_matrix = Matrix4::from_nonuniform_scale(transform.scale.x, -transform.scale.y, transform.scale.z);
+        let scale_matrix = match self.version.equals_or_above(2, 2) {
```

Comment:
Did they really flip the scale dimension direction? .... oh gravity ...

Reply:
Not sure, when I loaded everything was upside down, so I just turned the negative sign and it worked after removing the bounding box shift. I  think is a fix from the original method that was already upside down.